core/storage/classifier/gae_models_test.py

```diff
@@ -313,6 +337,42 @@ def test_get_deletion_policy(self) -> None:
             .get_deletion_policy(),
             base_models.DELETION_POLICY.NOT_APPLICABLE)
 
+    def test_get_export_policy(self) -> None:
+        sample_dict = base_models.BaseModel.get_export_policy()
+        sample_dict.update(
+            {'exp_id': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'exp_version': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'state_name': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'algorithm_ids_to_job_ids': base_models.EXPORT_POLICY.NOT_APPLICABLE
+            })
+        self.assertEqual(
+            classifier_models.StateTrainingJobsMappingModel.get_export_policy(),
+            sample_dict)
+
+    def test_get_model(self) -> None:
+        exp_id = 'exp_id1'
+        exp_version = 1
+        state_name = 'state_name1'
+        job_id = 'job_id1'
+        classifier_models.StateTrainingJobsMappingModel.create(
+            exp_id, exp_version, state_name, {'algorithm_id': job_id})
+
+        mapping = (
+            classifier_models.StateTrainingJobsMappingModel.get_model(
+                exp_id, exp_version, state_name))
+        # Ruling out the possibility of None for mypy type checking.
+        assert mapping is not None
+
+        self.assertEqual(mapping.exp_id, exp_id)
+        self.assertEqual(mapping.exp_version, 1)
+        self.assertEqual(mapping.state_name, state_name)
```

Comment:
How about the remaining field (the one with algorithm_id)?

Reply:
Actually, the `algorithm_id` field is a necessary argument for `.create.` but not required to get the model

Reply:
This doesn't address my question. If you want to check that the model is populated correctly, you should probably check all of its fields.

(Alternatively, if you're testing something different here, try to make the test name clearer about its intent. See the guidance on writing backend tests [here](https://github.com/oppia/oppia/wiki/Backend-tests#guidelines-for-writing-good-tests)).

Reply:
Done! Also, I've edited the name of another test that tested for `.get_models` but had the name 'test_get_model_..` since it was clashing the apt name for the `get_model` test.